homeassistant/components/wallbox/sensor.py

```diff
@@ -128,6 +129,12 @@ class WallboxSensorEntityDescription(SensorEntityDescription):
         name="Depot Price",
         precision=2,
     ),
+    CHARGER_ENERGY_PRICE_KEY: WallboxSensorEntityDescription(
```

Comment:
- Should this have a currency as its unit of measurement?
- Should this have a `state_class`?
- Should this have a `device_class` of `monetary`?

Reply:
Hi, the currency differs with the country the wallbox is being used in, therefore no specific unit of measurement is chosen here. I did add a state and device class as you suggested.

Reply:
That would be problematic in the case of the device class, as it requires a unit of measurement.

Reply:
Yes, I see (did not know that). How about this, I am now retrieving the currency from the API. Also, made a change in sensor, if it is monetary, return the API currency.